src/components/views/messages/ReactionsRowButton.tsx

```diff
@@ -106,21 +107,48 @@ export default class ReactionsRowButton extends React.PureComponent<IProps, ISta
 
         const room = this.context.getRoom(mxEvent.getRoomId());
         let label: string | undefined;
+        let customReactionName = "";
```

Comment:
```suggestion
        let customReactionName: string | undefined;
```

I think I'd prefer this but it's more of a personal preference